tests/storage/hostdisk.go

```diff
@@ -0,0 +1,31 @@
+package storage
+
+import (
+	"fmt"
+	"path/filepath"
+
+	k8sv1 "k8s.io/api/core/v1"
+
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/tests/framework/checks"
+	"kubevirt.io/kubevirt/tests/libpod"
+)
+
+const (
+	BinBash = "/bin/bash"
```

Comment:
since this is used only once, I don't see value in defining it, and certainly not exposing it with a capital `B`.

Reply:
Thanks for the comment, fixed.
We can use this constant from the `tests` package, instead of redefining it or copy the string over.

Reply:
I find this even worse, as it adds exposure to the clunky test/utils file. `/bin/bash` is a simple string, that has not changed in the last two decades. I don't think that it is worth to add a cross-package dependency just to avoid repeating it.

Constants should add a level of abstraction, to protect the user from details that may change. `BinBash` does not do this, hence I would prefer we never had it.

Reply:
`BinBash` is currently being used 19 times across 5 different packages (+ locally at `tests`), including in the same package `tests/storage`, so I assumed it would be ok to add one more reference to it without introducing any new cross-package dependency.
Are you suggesting to eliminate `BinBash` all across the board?

Reply:
> I assumed it would be ok to add one more reference to it without introducing any new cross-package dependency.

I understand, but I hope we can avoid it.

> Are you suggesting to eliminate BinBash all across the board?

Eventually, yes. If you `git log test/utils.go` you can see me removing public constants multiple times.

Reply:
ok, done for this instance.

should we just find+replace all `BinBash` occurrences to just `/bin/bash`, or we should have local constants of `binBash` in every package that uses `/bin/bash` more than once?

Reply:
> should we just find+replace all BinBash occurrences to just /bin/bash, or we should have local constants of binBash in every package that uses /bin/bash more than once?

I am not familiar with all the users of this constant. Maybe there's a better abstraction in several of  them.